bp_be/src/include/bp_be_internal_if_defines.vh

```diff
@@ -104,8 +106,13 @@
     logic                                   mem3_v;                                                \
     logic [rv64_eaddr_width_gp-1:0]         mem3_pc;                                               \
     logic                                   mem3_cache_miss_v;                                     \
+    logic                                   mem2_tlb_miss_v;                                       \
+    logic                                   mem3_tlb_miss_v;                                       \
     logic                                   mem3_exception_v;                                      \
     logic                                   mem3_ret_v;                                            \
+    logic                                   mem3_itlb_fill_v;                                      \
```

Comment:
These should go directly from bp_be_mem to bp_be_director, not through the calculator